server/src/main/java/org/elasticsearch/action/datastreams/PutDataStreamOptionsAction.java

```diff
@@ -7,7 +7,7 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-package org.elasticsearch.datastreams.options.action;
+package org.elasticsearch.action.datastreams;
```

Comment:
Was looking for where this was now used that it needed to be moved and I think I missed it. Where is this used now?

Reply:
It's for serverless.